hiddenlayer/sdk/version.py

```diff
@@ -1 +1 @@
-VERSION = "1.1.1"
+VERSION = "1.1.2"
```

Comment:
Presuming we decide not to make this a breaking change, I think the version should be `1.2.0`.  It's more than just a patch.  It is new functionality for this function.
```suggestion
VERSION = "1.2.0"
```

Reply:
done